semanticdb/semanticdb3/guide.md

```diff
@@ -266,6 +266,19 @@ be passed through Scalac in the form of `-P:semanticdb:<option>:<value>`
     </td>
     <td><code>on</code></td>
   </tr>
+  <tr>
+    <td><code>-P:semanticdb:md5:&lt;value&gt;</code></td>
+    <td>
+        <code>on</code>,<br/>
+        <code>off</code>
+    </td>
+    <td>
+      Specifies whether to save a hexadecimal formatted MD5 fingerprint of the source
+      file contents in <code>TextDocument.md5</code> (<code>on</code> for yes,
+      <code>off</code> for no).
+    </td>
+    <td><code>on</code></td>
```

Comment:
If this is default, let's disable text:on by default.

Reply:
What will that mean for metap? Should we support --sourceroot

Reply:
If there's no text, then we don't prettyprint identifiers. I suggest we don't introduce additional options just yet.